service/src/main/java/uk/gov/hmcts/reform/fpl/service/respondent/RespondentAfterSubmissionValidator.java

```diff
@@ -33,21 +35,29 @@ public class RespondentAfterSubmissionValidator {
     private final UserService userService;
 
     public List<String> validateLegalRepresentation(CaseData caseData) {
+        return validateLegalRepresentation(caseData, false);
+    }
+
+    public List<String> validateLegalRepresentation(CaseData caseData, boolean hideRespondentIndex) {
         List<String> errors = new ArrayList<>();
 
         List<Element<Respondent>> respondents = nullSafeList(caseData.getRespondents1());
 
         for (int i = 0; i < respondents.size(); i++) {
             Respondent respondent = respondents.get(i).getValue();
-            errors.addAll(legalRepresentationErrors(respondent, i + 1));
+            errors.addAll(legalRepresentationErrors(respondent, hideRespondentIndex ? -1 : (i + 1)));
         }
 
         return errors;
     }
 
     public List<String> validate(CaseData caseData, CaseData caseDataBefore) {
+        return validate(caseData, caseDataBefore, false);
+    }
 
-        List<String> errors = new ArrayList<>();
+    public List<String> validate(CaseData caseData, CaseData caseDataBefore, boolean hideRespondentIndex) {
+
+        LinkedHashSet<String> errors = new LinkedHashSet<>();
```

Comment:
double check why this can't just be a list

Reply:
The reason for using LinkedHashSet is to support the test case result. I have changed it back to using list and fixed the test case.